torchvision/prototype/transforms/_auto_augment.py

```diff
@@ -137,7 +137,8 @@ def _apply_image_or_video_transform(
         elif transform_id == "Posterize":
             return F.posterize(image, bits=int(magnitude))
         elif transform_id == "Solarize":
-            return F.solarize(image, threshold=magnitude)
+            bound = 1.0 if isinstance(image, torch.Tensor) and image.is_floating_point() else 255.0
```

Comment:
We need to change this to 

```suggestion
            bound = _FT._max_value(image) isinstance(image, torch.Tensor) else 255.0
```

in #6830.

Reply:
I noted this earlier at https://github.com/pytorch/vision/pull/6874#discussion_r1009457953

This is not supposed to happen yet though. Do you mean something else?

Reply:
Sorry, missed that. No, I mean exactly that.